packages/passport-interface/src/SubscriptionManager.ts

```diff
@@ -80,8 +80,8 @@ export class FeedSubscriptionManager {
    * This "refreshes" a feed. Existing feed errors are cleared, and new
    * ones may be detected.
    *
-   * Returns an array of actions without trying to apply them. Attempted
-   * application may lead to more errors.
+   * Returns the successful responses. Failures will be recorded in
+   * `this.errors` for display to the user.
```

Comment:
Nothing to do with this PR, but I failed to amend this comment in a previous PR and it was bugging me.